addons/neon/templates/componentdefinition-neon-pageserver.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  labels:
+    {{- include "neon.labels" . | nindent 4 }}
+  name: {{ include "neon-pageserver.componentDefName" . }}
+spec:
+  description: A neon pageserver component definition for Kubernetes
+  provider: KubeBlocks
+  scripts:
+    - name: neon-scripts
+      templateRef: neon-scripts
+      namespace: {{ .Release.Namespace }}
+      volumeName: scripts
+      defaultMode: 0555
+  vars: 
+    - name: NEON_STORAGEBROKER_HEADLESS
+      valueFrom:
+        serviceVarRef:
+          compDef: {{ include "neon-storagebroker.componentDefName" . }}
+          name: headless
+          optional: true
+          host: Optional
+    - name: NEON_STORAGEBROKER_POD_LIST
+      valueFrom:
+        componentVarRef:
+          compDef: {{ include "neon-storagebroker.componentDefName" . }}
+          optional: true
+          instanceNames: Optional
+  runtime:
+    containers:
+    - command: ["/scripts/pageserver_start.sh"]
+      image: {{ include "neon-pageserver.image" . }}
+      name: neon-pageserver
+      volumeMounts:
+      - mountPath: /usr/local/neon/pageserver
+        name: neon-pageserver
+      - name: scripts
+        mountPath: /scripts
+  serviceKind: neon-pageserver
+  serviceVersion: {{ .Values.componentServiceVersion }}
```

Comment:
is all the ComponentDefinitions has the same serviceVersion?

Reply:
They all use the same image, so the serviceVersion should be consistent?